app/controllers/event_disavowal_controller.rb

```diff
@@ -16,7 +16,7 @@ def new
 
   def create
     result = password_reset_from_disavowal_form.submit(password_reset_params)
-    analytics.event_disavowal_password_reset(**result.to_h)
+    analytics.event_disavowal_password_reset(user_id: disavowed_event.user_id, **result.to_h)
```

Comment:
`PasswordResetFromDisavowalForm` internally uses `BuildDisavowedEventAnalyticsAttributes` for its extra attributes, so we don't need to include this here again. One of the nice things about reusing `BuildDisavowedEventAnalyticsAttributes`.

```suggestion
    analytics.event_disavowal_password_reset(**result.to_h)
```